design/sketch.webidl

```diff
@@ -119,8 +111,31 @@ dictionary WebGPUTextureDescriptor {
     WebGPUTextureUsageFlags usage;
 };
 
+// Texture view
+enum WebGPUTextureViewDimension {
+    "1d",
+    "2d",
+    "2darray",
+    "cube",
+    "cubearray",
+    "3d"
+};
+
+dictionary WebGPUTextureViewDescriptor {
+    WebGPUTextureFormat format;
+    WebGPUTextureViewDimension type;
+    u32 baseMipLevel;
+    u32 levelCount;
+    u32 baseArrayLayer;
+    u32 layerCount;
+};
+
+interface WebGPUTextureView {
+};
+
 interface WebGPUTexture {
     WebGPUTextureView createTextureView(WebGPUTextureViewDescriptor desc);
```

Comment:
Maybe we want to have `createTextureView` be on the device so that `WebGPUTexture` is a opaque handle and that's it. WDYT?

Reply:
I have moved createTextureView to WebGPUDevice. PTAL, thanks!

Reply:
@kvark agreed that it doesn't make a big difference and there's no special benefits to `WebGPUTexture` being an empty interface. I don't really mind either way so maybe we can keep it in texture and bikeshed later?

@Jiawei-Shao sorry for the churn on this and https://dawn-review.googlesource.com/c/dawn/+/1580 can you rollback the changes?

Reply:
Done